src/fonduer/candidates/models/candidate.py

```diff
@@ -24,30 +24,27 @@ class Candidate(_meta.Base):
     """
 
     __tablename__ = "candidate"
+
+    #: The unique id for the ``Candidate``.
     id = Column(Integer, primary_key=True)
+
+    #: The type for the ``Candidate``, which corresponds to the names the user
+    #: gives to the candidate_subclasses.
     type = Column(String, nullable=False)
+
+    #: Which split the ``Candidate`` belongs to. Used to organize train/dev/test.
     split = Column(Integer, nullable=False, default=0, index=True)
 
     __mapper_args__ = {"polymorphic_identity": "candidate", "polymorphic_on": type}
 
     # __table_args__ = {"extend_existing" : True}
 
     def get_contexts(self):
```

Comment:
`Context` was renamed to `Doc` or `Document` in other parts. Maybe do the same here?

Reply:
Good point. This actually gets Mentions, so it should probably just be renamed to that.